pkg/plugins/plugindef/plugindef.cue

```diff
@@ -120,6 +120,24 @@ seqs: [
 					...
 				}
 
+				#ExtensionsLink: {
+					// Unique identifier of the plugin link extension
+					id:          string
+					description: string
+					// Relative path to the plugin
+					path: string
+					...
```

Comment:
The contents of this struct are so generic that i have to imagine this is doing all the heavy lifting - we're basically bypassing actually schematizing link extensions?

Reply:
hmm, are you looking for a way to lock this down even more?

Reply:
i didn't really grok the notion behind the link extension at first. It makes sense now - the `path` field is sufficient to inform the planned behavior for this extension point.